content/projects/data-science-specific/k-means-clustering/_index.md

```diff
@@ -28,14 +28,22 @@ title: K-Means Clustering Assignment
 
 ## Pre-requisites
 
-Read through the K-Means Tutorials at {{< contentlink path="topics/data-science-specific/clustering" >}} before starting this project.
+Read through the K-Means Tutorials at [content link](http://syllabus.africacode.net/topics/data-science-specific/clustering/) before starting this project.
```

Comment:
Please dont use absolute urls like this. We use contentlinks for a reason.  One reason is that so we can detect if a link breaks. 

Also, prerequisites should be listed in the frontmatter. This isn't the place for them.

Reply:
Sure

Reply:
I think Sheena's comment above still needs to be addressed - that link is already shown as a hard pre-requisite in the front matter, so maybe not necessary to show it twice?